lib/src/utils.dart

```diff
@@ -3,14 +3,76 @@
 // BSD-style license that can be found in the LICENSE file.
 library dartdoc.utils;
 
+final RegExp leadingWhiteSpace = new RegExp(r'^( *)[^ ]');
+const int kTabWidth = 8;
+
+/// Try to do something with tabs, by converting them to spaces.
+String convertTabs(String str) {
```

Comment:
do people use tabs in doc comments? Or, should we support them? I think we could reasonably just treat them as a single space whitespace char.

Reply:
good idea.  done.